### DISCUSSION TITLE
Hi!!

### DISCUSSION THREAD

**Discussion by liumOazed** [Cyfrin/foundry-full-course-cu#2485] [General]
Finished the 2 hr marks of all the pre coding basics. Now will start coding wish me luck guys. 💪

**Comment by EngrPips** [comment]
Best of luck, Friend.

**Comment by abdullahansari-yk** [comment]
All the very best

**Comment by chainflash** [comment]
> Finished the 2 hr marks of all the pre coding basics. Now will start coding wish me luck guys. 💪

Be proud of yourself

**Comment by Dardee001** [comment]
Happy coding my friend
